<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgDemo54</class>
 <widget class="QDialog" name="dlgDemo54">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Reportes en PyQT usando el QPainter y QtPrintSupport</string>
  </property>
  <widget class="QTableView" name="tblProducto">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>761</width>
     <height>401</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblTotalMensaje">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>460</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Total de Productos:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblTotalRegistros">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>460</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPreview">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Preview</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPrint">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnExportarPdf">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>101</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Exportar PDF</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
